Python/Pomodoro_Timer/Pomodoro.py

```diff
@@ -0,0 +1,152 @@
+import time
+
+# path of host file in windows
+host_path = r"C:\Windows\System32\drivers\etc\hosts"
+
+# URL of websites to block
+block_list = [
+    'www.facebook.com', 'facebook.com',
+    'www.youtube.com', 'youtube.com',
+    'www.gmail.com', 'gmail.com',
+    'www.instagram.com', 'instagram.com',
+    'www.twitter.com', 'twitter.com'
+]
+
+# redirecting above URLs to this localhost to ensure blocking
+redirect = "127.0.0.1"
+
+
+def block_websites():
+    """
+    The function will open the host file and add the block-list websites to
+    the file if it is not already present and redirect it to the localhost
+    for blocking
+    """
+
```

Comment:
Nitpicking here, but can we get rid of this blank line after docstring, the script is really well structured, this is really very impressive.